### PR TITLE
Documentation Review

### PR DESCRIPTION
**Pull request by rybons**
Rephrasing, typos, structuring etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Note that the examples as well as the code used to provision the necessary infra
 
 # Preparation
 
-You can make use of existing Automate, Jenkins, and Vault instances, for the sake of these examples. Skip forward to "\<tool\> Setup", otherwise, follow the infrastructure guide below.
+You can choose to make use of existing Automate, Jenkins, and Vault infrastructure when following these examples (in which case, skip forward to "\<tool\> Setup" sections). If you need to stand up this infrastructure yourself, follow the infrastructure guide below.
```
Comment (rybons) [inline README.md:15]: Maybe use links to the section headers here as `\<tool\> Setup` might confuse people.

File: README.md [hunk 2]
```diff
@@ -40,8 +40,8 @@ A few Habitat packages must be built before the infrastructure can be provisione
 - **infra-linux-hardening** is used to harden the servers and is loaded as part of the [base applications](habitat-plans/infra-linux-base-applications-with-hardening/Policyfile.rb)
 - **infra-linux-base-applications** is used to load base applications, including **inspec-linux-audit**
 - **consul** is used as a backend to Vault and is used in [infrastructure-terraform/vault](infrastructure-terraform/vault)
-- **vault** used in [infrastructure-terraform/vault](infrastructure-terraform/vault)
-- **jenkins** used in used in [infrastructure-terraform/jenkins](infrastructure-terraform/jenkins)
+- **vault** is used in [infrastructure-terraform/vault](infrastructure-terraform/vault)
```
Comment (rybons) [inline README.md:43]: Fixed some typos here but this section needs to be more specific about the two directions you can go:

If you choose not to build these packages under your own origin, you may not be able to do the 4th example (as you need to be able to push changes to these packages). Additionally, you may only need to build some of these packages (i.e. no need to build Jenkins under your own origin).

File: README.md [hunk 3]
```diff
@@ -292,7 +292,7 @@ Our first example consists of two individual pipelines - an operating system pip
 
 The [operating system pipeline](examples/example-1-server-provisioning/packer-pipeline) consists of a Jenkinsfile that makes use of Packer with the code found in the [packer](packer) folder. This job runs packer against the JSON image definition using the included variable file for CentOS. Note this may need to be adjusted if you are not using the `liftoff-modern-application-delivery` origin and the corresponding base OS package. This can be overriden in the job configuration by defining an environment variable `TF_VAR_variable_name`, so for `habitat_origin` this might be `TF_VAR_habitat_origin`.
 
-Due to the need for this image to exist before Jenkins is up (to deploy Jenkins), this pipeline does not need to be run but can be used as a starting point for a continuous operating system pipeline.
+Due to the need for this image to exist before Jenkins is up (to deploy Jenkins), an image was already created in section 2.4 (Virtual Machine Image Creation). This pipeline does not *need* to be run to proceed with examples, but can be used as a reference for what a continuous operating system pipeline that leverages Packer might look like.
```
Comment (rybons) [inline README.md:295]: Maybe link the `Virtual Machine Image Creation` section instead of the way I did it.

File: README.md [hunk 4]
```diff
@@ -282,7 +282,7 @@ Log in. If using the included terraform, this requires you SSH to the machine wi
 
 # Examples
 
-The examples are run through Jenkins making use of the Pipelines defined in [examples](examples). 
+The examples below are all executed through use of Jenkins Pipelines. The pipeline code can be found in [examples](examples).
```
Comment (rybons) [inline README.md:285]: This change is just rephrasing but the examples section in general might benefit from a statement about the practical use case statements for each example, such as:

## Example 1: Infrastructure Pipeline

_I want to be able to rebuild my base operating system using code_

---

## Example 2: Application Automation

_I want to be able to deploy updates to my applications using code_

File: README.md (hunk 1 above)
Comment (sirajrauff) [reply README.md:15]: Resolved in #2

File: README.md (hunk 2 above)
Comment (sirajrauff) [reply README.md:43]: Resolved in #2

File: README.md (hunk 4 above)
Comment (sirajrauff) [reply README.md:285]: Resolved in #2

File: README.md (hunk 3 above)
Comment (sirajrauff) [reply README.md:295]: Resolved in #2

**Review by sirajrauff — APPROVED** [review]

**Pull request merged**